src/hijackers/dynamicScript.ts

```diff
@@ -0,0 +1,26 @@
+import { execScripts } from 'import-html-entry';
+import { Freer } from '../interfaces';
+
+export default function hijack(proxy: Window): Freer {
+  const rawHtmlAppendChild = HTMLHeadElement.prototype.appendChild;
+
+  function wrapHtmlAppendChild<T extends Node>(this: any, child: T) {
+    if ((child as any).tagName !== 'SCRIPT') return rawHtmlAppendChild.call(this, child);
+
+    const script = (child as any) as HTMLScriptElement;
+    if (script.src) {
+      return execScripts('', [script.src], proxy);
```

Comment:
pass `null` to execScripts method may be more semantical than an empty string

Reply:
enen, i updated